index.html

```diff
@@ -210,7 +210,7 @@ <h1 class="featureSpeaker_text">Feature Speakers</h1>
 
     <!-- partners section -->
 
-    <section class="partner-sec">
+    <section id="partner-sec" class="partner-sec">
```

Comment:
- [ ] You can consider reducing the page between the `logo` on the left and the `text`  on the right, 🤔  I think you could use `justify-content: space-around;` or `center` instead of using `justify-content: space-between;`  🖌️  
- [ ] Kindly make this to your About page too :wink:

- I know you can do it :muscle: 😄 


- [Cindy Shin](https://www.behance.net/adagio07)[Cindy Shin](https://www.behance.net/adagio07) design:

![foot1](https://user-images.githubusercontent.com/79058364/168645688-6f1736a4-f127-4f6f-8eca-363f70ad7904.png)
- Your design:

![footer52](https://user-images.githubusercontent.com/79058364/170738315-5644cbb6-8217-422d-8ff6-0e726afb277f.png)